src/components/file/icons.ts

```diff
@@ -125,5 +125,5 @@ const filetypeIconTable: Record<string, string> = {
 };
 
 export function getIconForFile(extension: string): string {
-    return filetypeIconTable[extension] || DEFAULT_ICON;
+    return filetypeIconTable[extension.toLowerCase()] || DEFAULT_ICON;
```

Comment:
I would suggest that we test this with a file that has no filename extension, just to make sure that doesn't result in an error.

Reply:
Ah, I found it. I'll push a fix for the colours 👍

Reply:
⚡

Reply:
@anderssonjohan If you're happy with my update, I'll squash and merge this 👍

Reply:
I thought you were expecting me to squash so I've already done that. Just need your accept on the review again since it became stale when I pushed.